openstack/profiles/prodstack6

```diff
@@ -3,7 +3,15 @@ source ~/novarc
 # this is currently assumed to be a /25 network
 EXT_SUBNET=subnet_${OS_USERNAME}-psd-extra
 CIDR=`openstack subnet show $EXT_SUBNET -c cidr -f value`
-ABC=${CIDR%.0*}
+# We reserve the last 64 of the /25 to FIP.
+# We get the last usable IP on the subnet
+FIP_RANGE_LASTIP=`openstack subnet show subnet_stg-reproducer-dnegreira-psd-extra -c allocation_pools -f json | jq -r '.allocation_pools[0].end'`
```

Comment:
This should be `$EXT_SUBNET` instead of `subnet_stg-reproducer-dnegreira-psd-extra`

Reply:
Ops :)

Reply:
fixed on https://github.com/canonical/stsstack-bundles/pull/205/commits/bd4bee6de926f751b33d33c387b18ec8b7de627a by @nicolasbock